src/out_hoi4/states/out_state.cpp

```diff
@@ -21,6 +21,16 @@ void out::OutputState(std::string_view output_name, const hoi4::State& state)
    state_history << "state = {\n";
    state_history << fmt::format("\tid = {}\n", state_number);
    state_history << fmt::format("\tname = \"STATE_{}\"\n", state_number);
+   state_history << "\tmanpower = 1000\n";
```

Comment:
hardcoding outputs in out_functions? I hope you remember to fix this. :)

Reply:
#59 and #60 :-)